### PR TITLE
Add bitnami message

### PR DESCRIPTION
**Pull request by tompizmor**
This PR changes the index page to show a Bitnami message redirecting users to the sample-mean repository so they can check how to deploy their own applications.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -72,9 +72,11 @@ <h1>I'm a Todo-aholic <span class="label label-info">{{ todos.length }}</span></
 			</div>
 		</div>
 
-		<div class="text-center text-muted">
-			<p>A demo by <a href="http://scotch.io">Scotch</a>.</p>
-			<p>Read the <a href="http://scotch.io/tutorials/javascript/creating-a-single-page-todo-app-with-node-and-angular">tutorial</a>.</p>
+                <img src="img/bitnami.png" class="img-responsive" style="width:15%;margin:auto;padding-bottom:15px" alt="Bitnami Logo">
+
```
Comment (jotadrilo) [inline public/index.html]: I would not remove the reference to `http://scotch.io`.
Let's add it as a footer note or similar

Comment (dbarranco) [reply public/index.html]: +1

**Comment by dbarranco** [comment]
As the utrera-guy pointed, we could add Bitnami-aholic instead of Todo-aholic.

Line to modify:
https://github.com/bitnami/sample-mean/blob/ab1e2cddb81535bf9fd1b4b2e9d03f697414ed0f/public/index.html#L36

File: public/index.html [hunk 2]
```diff
@@ -71,15 +71,18 @@ <h1>I'm a Todo-aholic <span class="label label-info">{{ todos.length }}</span></
...
+		<div class="text-center text-muted">
+			<p>A demo by <a href="http://scotch.io">Scotch</a>.</p>
+			<p>Read the <a href="http://scotch.io/tutorials/javascript/creating-a-single-page-todo-app-with-node-and-angular">tutorial</a>.</p>
+		</div>
+               
+		<img src="img/bitnami.png" class="img-responsive" style="width:15%;margin:auto;padding-bottom:15px;padding-top:15px" alt="Bitnami Logo">
 
                 <div class="text-center text-muted">
-			<p>This is an example Mean application hosted in <a href="https://github.com/bitnami/sample-mean">GitHub</a> and based on the <a href="https://github.com/bitnami/sample-mean">scotch node-todo application</a>.</p>
+			<p>This is an example MEAN application hosted in <a href="https://github.com/bitnami/sample-mean">GitHub</a> and based on the <a href="https://github.com/bitnami/sample-mean">scotch node-todo application</a>.</p>
```
Comment (jotadrilo) [inline public/index.html]: I think we should refer to:

```
and based on the <a href="https://github.com/scotch-io/node-todo">scotch node-todo application
```

Comment (tompizmor) [reply public/index.html]: yes, I forgot to update the link

**Pull request merged**